lib/Echidna/Processor.hs

```diff
@@ -76,7 +85,7 @@ runSlither fp args = let args' = ["--ignore-compile", "--print", "echidna", "--j
       (ec, out, _) <- readCreateProcessWithExitCode (proc path $ args' |> fp) {std_err = Inherit} ""
       case ec of
         ExitSuccess -> return $ procSlither out
-        ExitFailure _ -> return [] --we can make slither mandatory using: throwM $ ProcessorFailure "slither" err
+        ExitFailure _ -> throwM $ ProcessorFailure "slither" "You should install it using 'pip3 install slither-analyzer --user'"
```

Comment:
shouldn't this happen when `mp` is `Nothing`? This case is when slither was found but running it failed with non 0 code.

Reply:
Good catch. Fixed.